src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+    color: $title-text-on-hover;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px 36px;
+  }
+
+  &__product {
+    display: flex;
+    flex-direction: column;
+    padding: 0 16px;
+    margin-bottom: 16px;
+
+    &-name {
+      margin: 0;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &-code {
+      color: $secondary-font-color;
+      margin: 0;
+      padding-top: 4px;
+      font-size: 10px;
+      font-weight: 300;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 16px;
+    margin-bottom: 16px;
+    color: $main-font-color;
+
+    &-text {
+      margin: 0;
```

Comment:
Why do you need to set `margin: 0` in every class-selector that you set on `p` tag? I think you can just remove that default margin for p tag in one place, somewhere in main.scss file or create separate file like normalize.scss